packages/codegen/config-json-schema.json

```diff
@@ -47,7 +47,10 @@
                     "superstruct": { "type": "string" },
                     "zodSchema": { "type": "string" },
                     "type": { "type": "string" },
-                    "serde": { "type": "string" }
+                    "serde": { "type": "string" },
+                    "stiType": { "type":  "string" },
+                    "notNull": { "type": "boolean" },
+                    "stiDiscriminator": { "type": "object" }
```

Comment:
Thanks for updating this! ...now I have to remember how to upload it to the schema registry in the sky that webstorm/vscode use...